src/agent.ts

```diff
@@ -57,9 +58,10 @@ export async function createAgent() {
   const storageConfig = POSTGRES_HOST ? askarPostgresConfig : undefined
 
   const walletConfig: WalletConfig = {
-    id: WALLET_NAME,
+    id: WALLET_ID,
     key: WALLET_KEY,
     storage: storageConfig,
+    keyDerivationMethod: KeyDerivationMethod.Raw,
```

Comment:
If using RAW as the KDF, we should add some hints to the README on how you can generate a key.